### PR TITLE
update finalzier to CRD doc

### PR DESCRIPTION
**Pull request by stuggi**
Depends-On: https://github.com/openstack-k8s-operators/lib-common/pull/519
Depends-On: https://github.com/openstack-k8s-operators/mariadb-operator/pull/231

### CODE REVIEW

File: tests/kuttl/common/assert-sample-deployment.yaml [hunk 1]
```diff
@@ -75,7 +75,7 @@ apiVersion: heat.openstack.org/v1beta1
 kind: HeatCfnAPI
 metadata:
   finalizers:
-    - HeatCfnAPI
+    - openstack.org/heatcfnaPI
```
Comment (abays) [inline tests/kuttl/common/assert-sample-deployment.yaml:78]: Wondering if `heatcfnaPI` might have wrong case

Comment (stuggi) [reply tests/kuttl/common/assert-sample-deployment.yaml:78]: right thats wrong, thanks for the pointer

Comment (stuggi) [reply tests/kuttl/common/assert-sample-deployment.yaml]: fixed

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**